notes_02-02-2016.md

```diff
@@ -16,3 +16,7 @@ The different parts should be based on our own insights and how the different pr
 He also said it was important to have peer review ensure that everyone is contributing to the project.
```

Comment:
@softwaregast seems a good plan to have one person to write a part. I'm fine with being a Researcher. Shall we make groups tomorrow during the online meeting?